### PR TITLE
Sage 300: Shared Shimmers Component - 1.1

### PR DESCRIPTION
**Pull request by anishfyle**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new `ShimmersComponent` to enhance visual feedback during data loading processes across various components in the dashboard and export sections.

- **Refactor**
  - Replaced multiple instances of inline conditional rendering with the reusable `ShimmersComponent` to improve code maintainability and reduce duplication.

- **Style**
  - Updated user interface elements to include loading shimmers for a better user experience during data fetching operations.

### CODE REVIEW

**Comment by ashwin1111** [comment]
@coderabbitai review

File: src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html [hunk 1]
```diff
@@ -34,8 +34,5 @@
 </p-table>
 
 <ng-template #shimmers>
-    <tr *ngFor="let _ of [0,1,2,3,4,5,6]">
-        <td *ngIf="isLoading"><p-skeleton width="150px" height="28px" shape="rectangle"></p-skeleton></td>
-        <td *ngIf="isLoading"><p-skeleton width="250px" height="28px" shape="rectangle"></p-skeleton></td>
-    </tr>
+    <app-shimmers [isMappingTableShimmers]="true"></app-shimmers>
```
Comment (ashwin1111) [inline src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html:37]: how is the visibility controlled? showing / hiding

**Review by ashwin1111 — APPROVED** [review]

File: src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html (hunk 1 above)
Comment (anishfyle) [reply src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html:37]: have passed on isLoading to isMappingTableShimmers, from the generic-mapping-table

Comment (ashwin1111) [reply src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html]: this is generic-mapping-table no?

Comment (ashwin1111) [reply src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html]: got it

Comment (anishfyle) [reply src/app/shared/components/helper/mapping/generic-mapping-table/generic-mapping-table.component.html]: https://github.com/fylein/fyle-integrations-app/pull/344/commits/e54d08364daec52231577cb3b11ac038ec990d0e

**Pull request merged**